CHANGELOG.md

```diff
@@ -7,27 +7,133 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-## [0.5.5] - 03-12-20
+### Added
```

Comment:
If you want to merge into the default branch it means those changes belongs to 0.7.0, not Unreleased.